src/main/java/hudson/plugins/ec2/EC2Cloud.java

```diff
@@ -204,6 +215,25 @@ public EC2PrivateKey resolvePrivateKey(){
         return null;
     }
 
+    /* visible for testing */
+    @CheckForNull
+    public static EC2PrivateKey fetchPrivateKeyFromDisk() {
+        return fetchPrivateKeyFromDisk(System.getProperty(SSH_PRIVATE_KEY_FILEPATH, ""));
+    }
+
+    @CheckForNull
+    public static EC2PrivateKey fetchPrivateKeyFromDisk(String filepath) {
```

Comment:
Please, move these two methods to `EC2PrivateKey` where they make more sense. And doing so, you could even simplify the name: `fetchFromDisk` as it is clear when calling `EC2PrivateKey.fetchFromDisk`